### PR TITLE
650 dont export methods just for testing langchaints

### PR DESCRIPTION
**Pull request by gsproston-scottlogic**
## Description

Stopped exporting methods in langchain.ts that were only exported for testing.

## Notes

- Ended up removing the unit test file as it was only testing the methods in question, which overall were covered by tests in the integration test file.

## Checklist

Have you done the following?

- [x] Linked the relevant Issue
- [x] Added tests
- [x] Ensured the workflow steps are passing

### CODE REVIEW

**Review by heatherlogan-scottlogic — APPROVED** [review]
all looks good to me

**Review by dhinrichs-scottlogic — APPROVED** [review]
looks good to me, but feel free to wait for a more experienced dev to approve if you want 😄

**Pull request merged**
